.github/workflows/contributors.yml

```diff
@@ -46,7 +46,10 @@ jobs:
       # The lint workflow is not important for this type of PR, skip it and pretend it was successful:
       - name: 'Get the latest commit hash from the contributors-update branch'
         id: commit-data
-        run: 'echo "::set-output name=head_sha::$(git rev-parse contributors-update)"'
+        run: |
+          git pull
+          git checkout contributors-update
+          echo "::set-output name=head_sha::$(git rev-parse contributors-update)"
```

Comment:
The relevant commit should have been committed locally to be pushed to the remote `contributors-update` branch. I believe `git rev-parse @` would work just as well here, without needing to pull.

I was writing a response to you with that suggested fix, but forgot to send it prior to working on a PR that implemented it.

Reply:
I tried that before. Without `git pull`, `git checkout contributors-update` fails.

Reply:
> Not removing `git pull `

Doesn't the PyGithub API used by the `add-contributors` action not commit locally to push to the remote branch? It seems odd that a `git pull` would be required if the commit pushed is the same commit SHA locally and remote?

Something else perhaps is going on that I missed/misunderstood?

Reply:
I cannot help you with that. I don't know how `add-contributors` action works in detail (not very familiar with python).